rope/contrib/autoimport/parse.py

```diff
@@ -45,7 +46,7 @@ def get_names_from_file(
 ) -> Generator[PartialName, None, None]:
     """Get all the names from a given file using ast."""
     try:
-        root_node = ast.parse(module.read_bytes())
+        root_node = rope.base.ast.parse(module.read_bytes())
```

Comment:
As far as I am aware, this should use the stdlib ast parser

Reply:
@bagel897 Thanks for you comment! `rope.base.ast.parse` wraps `stdlib.ast.parse`: it regularizes the `source` argument and catches two exceptions. These seem like reasonable things to do.

@lieryan would like to replace the one remaining call to `stdlib.ast.parse` with a call to `rope.base.ast.parse`.

I see two remaining tasks before marking this PR as ready for review:

1. Get PR #633 accepted, so that `rope.base.ast` will define only two names: `rope_parse` (see below) and `RopeNodeVisitor`.

2. Submit a new PR that does the following:

- Rename `rope.base.ast.parse` to `rope_parse`.
- Remove all qualifications for `rope_parse` by adding
  `from rope.base.ast import rope_parse` as needed.
- Replace `ast.RopeNodeVisitor` with `RopeNodeVisitor`,
  adding  `from rope.base.ast import RopeNodeVisitor` as needed.

The new PR will simplify the appearance of the code, remove all ambiguity between the two `parse` methods, and reduce the diffs of *this* PR.